SurveyApp/Extensions/Date/DateFormatter+Format.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DateFormatter+Format.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/21/21.
+//
+
+import Foundation
+
+extension DateFormatter {
+
+    static let date = DateFormatter(with: "EEEE, MMMM d")
```

Comment:
The name is quite confused with `Date`, but in fact it's `DateFormatter`
```suggestion
    static let dateFormatter = DateFormatter(with: "EEEE, MMMM d")
```